### PR TITLE
[OSCD] Modify powershell_malicious_commandlets.yml to leverage ScriptBlock logging feature

### PR DESCRIPTION
**Pull request by bczyz1**
OSCD Sprint 2 Atomic Red Team Windows - #1013
Task 4 - T1056.001: Keylogging

Coverage for this ART test has already been achieved via the following rule https://raw.githubusercontent.com/Neo23x0/sigma/oscd/rules/windows/powershell/powershell_malicious_commandlets.yml

However, I am opening this pull request as a recommendation to make use of Script Block logging feature (EventID 4104) instead of keywords. This change also allows to use `contains` YAML modifier in place of explicit wildcard character `*` which is not supported by some Sigma backends.

### CODE REVIEW

**Comment by e6e6e** [comment]
Correct fixes. Thanks for your observation! )
@yugoslavskiy Please check it.

File: rules/windows/powershell/powershell_malicious_commandlets.yml [hunk 1]
```diff
@@ -10,110 +10,114 @@ tags:
     - attack.t1086  #an old one
 author: Sean Metcalf (source), Florian Roth (rule)
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_malicious_commandlets.yml:11]: Hello @bczyz1 !  I believe you can put there your name and oscd.community with `(update)` (:

Comment (bczyz1) [reply rules/windows/powershell/powershell_malicious_commandlets.yml]: Hey @yugoslavskiy, thanks, wasn't sure about that and decided to play it safe 😃 authors appended in e90f91b89ee905897d764de581a1e7ae000390ce, cheers!

**Comment by yugoslavskiy** [comment]
Hello @bczyz1! Great job! Thank you very much (:

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**